cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -436,7 +436,15 @@
 
     public static final String DATA_STORAGE_IMPLEMENTATION = "data.storage.implementation";
     public static final String DATA_STORAGE_NOSQL = "nosql";
-    public static final String DATA_STORAGE_SQL = "postgressql";
+    public static final String DATA_STORAGE_SQL = "postgresql";
+    public static final String DATA_STORAGE_SQL_DRIVER_DIRECTORY = "data.storage.sql.driver.directory";
+    public static final String DATA_STORAGE_SQL_JDBC_DRIVER_NAME = "data.storage.sql.jdbc.driver.name";
+
+    // the jdbc connection related properties should be from cdap-security.xml
+    public static final String DATA_STORAGE_SQL_JDBC_CONNECTION_URL = "data.storage.sql.jdbc.connection.url";
```

Comment:
Does connection URL have any sensitive information to add it to sConf?

Reply:
The username and password can be passed using the jdbc url, and it can contain the db name and connection instance name.

Reply:
We can document that username and password should only be specified in sConf if it needs to be secure. We want non-secure properties in cConf so that everyone can view it.